src/coord/src/catalog/builtin.rs

```diff
@@ -1261,14 +1263,19 @@ pub const PG_TYPE: BuiltinView = BuiltinView {
         0
     )
         AS typarray,
+    -- Unfortunately we cannot use 'array_in'::pg_catalog.regproc, so hard code the
+    -- array_in oid.
```

Comment:
Wait, why not?

Reply:
Unsure. My best guess is initialization order? The text -> regproc conversion uses pg_proc which maybe doesn't exist yet? I could attempt to track it down.

Reply:
For example, using the text version here works after mz has initialized.